src/rules/unit-blacklist/README.md

```diff
@@ -49,3 +49,48 @@ a { height: 100vmin; }
 ```css
 a { animation: animation-name 5s ease; }
 ```
+
+## Optional Options
+
+### `ignoreProperties: { unit: ["property", "/regex/"] }`
+
+Ignore the specified unites in the specified properties.
```

Comment:
How about this: `Ignore units in the values of declarations with the specified properties.`